Code/ForceField/MMFF/testMMFFForceField.cpp

```diff
@@ -1275,14 +1286,15 @@ int mmffValidationSuite(int argc, char *argv[]) {
                 std::stringstream diff;
                 diff << "Torsional: found a difference\n";
                 if (haveLine) {
-                  diff << "Expected:\n" << rdkLine << "\nFound:\n" << refLine
-                       << "\n";
+                  diff << "Expected:\n"
+                       << rdkLine << "\nFound:\n"
+                       << refLine << "\n";
                 }
                 errorMsg += diff.str();
               }
-              delete rdkTorsionInstanceVec[j];
-              delete refTorsionInstanceVec[j];
             }
+            BOOST_FOREACH (auto &ti, rdkTorsionInstanceVec) { delete ti; }
```

Comment:
better to do this with:
```suggestion
            for (auto &ti : rdkTorsionInstanceVec) { delete ti; }
```

Reply:
Ok.